eva-accession-core/src/main/java/uk/ac/ebi/eva/accession/core/SubmittedVariant.java

```diff
@@ -149,12 +163,13 @@ public boolean equals(Object o) {
                 Objects.equals(projectAccession, that.projectAccession) &&
                 Objects.equals(contig, that.contig) &&
                 Objects.equals(referenceAllele, that.referenceAllele) &&
-                Objects.equals(alternateAllele, that.alternateAllele);
+                Objects.equals(alternateAllele, that.alternateAllele) &&
+                Objects.equals(createdDate, that.createdDate);
```

Comment:
Are we going to do any comparison of SubmittedVariant objects anywhere in the code? I was thinking in this possible problem:
- We process a VCF and we store the accession in the DB. 
- We process the same VCF again (maybe it wasn't complete the first time, maybe is a different process that takes a VCF and return the accessions from the DB...)
- We compare both objects and are different because the one read from the VCF has created date null. But they are really the same submitted variant.

If something like that could happen, then we should remove created date from equals and toHash.

Reply:
They won't be compared like this to decide about its insertion (or not insertion). For that there's this class, that selects the fields to identify the object https://github.com/EBIvariation/eva-accession/blob/master/eva-accession-core/src/main/java/uk/ac/ebi/eva/accession/core/SubmittedVariantModelSummaryFunction.java